cli/central-client/src/main/java/org/ballerinalang/central/client/CentralAPIClient.java

```diff
@@ -195,23 +199,23 @@ public CentralAPIClient(String baseUrl, Proxy proxy, String proxyUsername, Strin
      */
     public Package getPackage(String orgNamePath, String packageNamePath, String version, String supportedPlatform,
             String ballerinaVersion) throws CentralClientException {
-        String packageSignature = orgNamePath + "/" + packageNamePath + ":" + version;
+        String packageSignature = orgNamePath + SEPARATOR + packageNamePath + ":" + version;
         Optional<ResponseBody> body = Optional.empty();
         OkHttpClient client = this.getClient();
         try {
-            String resourceUrl = "/" + PACKAGES + "/" + orgNamePath + "/" + packageNamePath;
+            String resourceUrl = SEPARATOR + PACKAGES + SEPARATOR + orgNamePath + SEPARATOR + packageNamePath;
```

Comment:
`SEPARATOR + PACKAGES + SEPARATOR` seems to be repeated extensively. Shall we create a final variable to hold that?

Reply:
Fixed with fc2c8abcedf